tests/amr/data/field/coarsening/coarsening_utils.py

```
@@ -0,0 +1,292 @@
+                        if qty == "Bx":
+                            coarseData[
+                                coarseLocalIndexX, coarseLocalIndexY, coarseLocalIndexZ
+                            ] = 0.25 * (
+                                fineData[fineIndexX, fineIndexY, fineIndexZ]
+                                + fineData[fineIndexX, fineIndexY + 1, fineIndexZ]
+                                + fineData[fineIndexX, fineIndexY, fineIndexZ + 1]
+                                + fineData[fineIndexX, fineIndexY + 1, fineIndexZ + 1]
+                            )
+
+                    elif not is_primal[0] and is_primal[1] and not is_primal[2]:
+                        if qty == "By":
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)